src/stream/src/common/table/state_table.rs

```diff
@@ -212,6 +212,11 @@ where
             .expect("non-replicated state store should start immediately.")
             .expect("non-replicated state store should not wait_for_epoch, and fail because of it.")
     }
+
+    /// Wait until the epoch is committed and its data is ready to read.
+    pub async fn wait_epoch(&self, epoch: u64) -> StorageResult<()> {
```

Comment:
Actually no need to add `wait_epoch` on `LocalStateStore`. We can just store the original `StateStore` in the `StateTable` and then call the `try_wait_epoch` of `StateStore`.